views/adminPartials/siteSettingsTabContent.php

```diff
@@ -21,27 +21,24 @@
             <h3>Site Color</h3>
         </div>
         <div class="card-body">
-            <form class="container">
-
                 <!-- color 1 -->
                 <div class="form-group">
                     <label for="color1">Select your color1:</label>
```

Comment:
Need to replace naming convention to more descriptive names such as text color, navbar color, and footer color.